hawkbit-rest/hawkbit-mgmt-resource/src/test/java/org/eclipse/hawkbit/mgmt/rest/resource/MgmtTargetResourceTest.java

```diff
@@ -1539,6 +1540,44 @@ public void getControllerEmptyAttributesReturnsNoContent() throws Exception {
                 .andDo(MockMvcResultPrinter.print()).andExpect(status().is(HttpStatus.NO_CONTENT.value()));
     }
 
+    @Test
+    @Description("Request update of Controller Attributes.")
+    public void triggerControllerAttributesUpdate() throws Exception {
```

Comment:
consider separating parts of this method out into separate private methods which are annotated with the Step annotation... improves the readability of our test reports...

Reply:
done :+1: